apps/condo/domains/ticket/hooks/useTicketTableFilters.tsx

```diff
@@ -0,0 +1,390 @@
+    getNumberFilter,
+    getStringContainsFilter,
+    getTicketAttributesFilter,
+} from '@condo/domains/common/utils/tables.utils'
+
+import { TicketSource, TicketStatus } from '../utils/clientSchema'
+import { searchEmployeeUser, searchOrganizationDivision, searchOrganizationProperty } from '../utils/clientSchema/search'
+import { useModalFilterClassifiers } from './useModalFilterClassifiers'
+
+export function useTicketTableFilters (): Array<FiltersMeta<MeterReadingWhereInput>>  {
+    const intl = useIntl()
+    const EmergencyMessage = intl.formatMessage({ id: 'Emergency' }).toLowerCase()
+    const NumberMessage = intl.formatMessage({ id: 'ticketsTable.Number' })
+    const PaidMessage = intl.formatMessage({ id: 'Paid' }).toLowerCase()
+    const DateMessage = intl.formatMessage({ id: 'Date' })
+    const StatusMessage =  intl.formatMessage({ id: 'Status' })
+    const DescriptionMessage = intl.formatMessage({ id: 'Description' })
+    const AddressMessage = intl.formatMessage({ id: 'field.Address' })
+    const EnterAddressMessage = intl.formatMessage({ id: 'pages.condo.meter.EnterAddress' })
+    const UserNameMessage = intl.formatMessage({ id: 'filters.UserName' })
+    const ExecutorMessage = intl.formatMessage({ id: 'field.Executor' })
+    const StartDateMessage = intl.formatMessage({ id: 'pages.condo.meter.StartDate' })
+    const EndDateMessage = intl.formatMessage({ id: 'pages.condo.meter.EndDate' })
+    const SourceMessage = intl.formatMessage({ id: 'field.Source' })
+    const SectionMessage = intl.formatMessage({ id: 'pages.condo.property.section.Name' })
+    const FloorMessage = intl.formatMessage({ id: 'pages.condo.property.floor.Name' })
+    const UnitMessage = intl.formatMessage({ id: 'field.FlatNumber' })
+    const EnterPhoneMessage = intl.formatMessage({ id: 'pages.condo.ticket.filters.EnterPhone' })
+    const ClientPhoneMessage = intl.formatMessage({ id: 'pages.condo.ticket.filters.ClientPhone' })
+    const AssigneeMessage = intl.formatMessage({ id: 'field.Responsible' })
+    const SelectMessage = intl.formatMessage({ id: 'Select' })
+    const PlaceClassifierLabel = intl.formatMessage({ id: 'component.ticketclassifier.PlaceLabel' })
+    const CategoryClassifierLabel = intl.formatMessage({ id: 'component.ticketclassifier.CategoryLabel' })
+    const DivisionLabel = intl.formatMessage({ id: 'pages.condo.ticket.filters.Division' })
+    const EnterUnitNameLabel = intl.formatMessage({ id: 'pages.condo.ticket.filters.EnterUnitName' })
+    const AttributeLabel = intl.formatMessage({ id: 'pages.condo.ticket.filters.Attribute' })
+    const AuthorMessage = intl.formatMessage({ id: 'pages.condo.ticket.filters.Author' })
+    const EnterFullNameMessage = intl.formatMessage({ id: 'pages.condo.ticket.filters.EnterFullName' })
+
+    const numberFilter = getNumberFilter('number')
+    const dateRangeFilter = getDayRangeFilter('createdAt')
+    const statusFilter = getFilter(['status', 'id'], 'array', 'string', 'in')
```

Comment:
you can move out of the function all filter generators created by `getFilter`
and also it's not obvious, when reading those variable names, that all those `xxxFilter` are, actually, functions and can be called. I think we have some kind of naming problem here.

Reply:
handed out the generators for the hook, with the naming it will be necessary to agree with @SavelevMatthew